assembl/alembic/versions/95750e0267d8_extract_taxonomies.py

```diff
@@ -0,0 +1,53 @@
+"""extract taxonomies
+
+Revision ID: 95750e0267d8
+Revises: cc4266c17ceb
+Create Date: 2018-02-06 09:06:58.932766
+
```

Comment:
I just added a alembic step on develop, you will need to rebase and use Revises: ae1da08a88b0